components/swatch/index.css

```diff
@@ -265,18 +266,11 @@
 	display: none;
 	pointer-events: none;
 
-	color: var(--highcontrast-swatch-disabled-icon-color, var(--mod-swatch-disabled-icon-color, var(--spectrum-swatch-disabled-icon-color)));
-	stroke: var(--highcontrast-swatch-disabled-icon-color, var(--mod-swatch-disabled-icon-color, var(--spectrum-swatch-disabled-icon-color)));
-
 	/* Icon fill color */
-	path:first-child {
-		fill: var(--highcontrast-swatch-disabled-icon-color, var(--mod-swatch-disabled-icon-color, var(--spectrum-swatch-disabled-icon-color)));
-	}
+	color: var(--highcontrast-swatch-disabled-icon-color, var(--mod-swatch-disabled-icon-color, var(--spectrum-swatch-disabled-icon-color)));
 
 	/* Icon outline color - does not need a highcontrast token because the icon fill color provides contrast. */
-	path:last-child {
-		fill: var(--mod-swatch-icon-border-color, var(--spectrum-swatch-icon-border-color));
-	}
```

Comment:
Previous svg markup for the cancel/disabled icon appears to have two paths, see the last [build](https://pr-3366--spectrum-css.netlify.app/swatch) of our deprecated docs site in November--our current svg icon only utilizes one path, so these styles don't work as intended.